Dockerfile

```diff
@@ -0,0 +1,9 @@
+FROM golang:1.14
```

Comment:
use multi stage dockerfiles pls

Reply:
Hmm, distroless images. That's interesting 😃